resources/assets/js/components/FriendDetails.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <button type="button" class="btn btn-primary btn-xs" @click="showDetails(friend)">Details</button>
+
+    <div id="{{ friend.username }}" class="hide">
+        <div class="friend-details-modal">
+            <img v-bind:src="friend.avatar_url" class="img-circle" height="160" />
+            <p><span>Name:</span> {{ friend.name }}</p>
+            <span v-show="friend.location && friend.location.length > 0">
+                <p><span>Location: </span> {{ friend.location }}</p>
+            </span>
+            <span v-show="friend.url && friend.url.length > 0">
+                <p><span>Url: </span><a href=\"" + friend.url + "\">{{ friend.url_display }}</a></p>
+            </span>
+            <span v-show="friend.description && friend.description.length > 0">
+                <p>{{ friend.description }}</p>
+            </span>
+        </div>
+    </div>
+</template>
+
+<script>
+    export default {
+        props: ['friend'],
+
+        methods: {
+            showDetails: function(friend) {
+                var $html = document.getElementById(friend.username).innerHTML;
```

Comment:
This is an interesting way to do this. I feel like there's a more vue-friendly way? Probably using v-el or whatever. But, TBH, this works, so, whatever. :)